pkgs/by-name/pa/payme/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "payme";
+  version = "v1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jovandeginste";
+    repo = pname;
```

```suggestion
    repo = "payme";
```

Do not parameterize `pname`.